lib/data/datasources/remote/storage_service/firestore_service.dart

```diff
@@ -0,0 +1,6 @@
+abstract class FirestoreService {
```

Comment:
firestore is purely about firebase and google service, the meaning of abstraction, take it out to StorageService as an abstraction, because firestore is a server on which data is stored and we will have several implementations from it, local and firestore

Reply:
i dont understand u,we gonna discuss it in call